src/app/news/news.component.html

```diff
@@ -7,7 +7,7 @@
     <mat-toolbar-row class="primary-color font-size-1"></mat-toolbar-row>
   </mat-toolbar>
   <div class="view-container view-full-height">
-    <div class="new-post-form">
+    <div *planetAuthorizedRoles="'learner'" class="new-post-form">
```

Comment:
@coder8102 `<div *ngIf="currentUser.isUserAdmin || currentUser.roles.length" class="news-post-form">` will work for this case.